rubix-bookkeeper/src/main/java/com/qubole/rubix/bookkeeper/LocalDataTransferServer.java

```diff
@@ -106,7 +106,8 @@ public void run()
                 log.info("Stopping Local Transfer server");
             }
             catch (IOException e) {
-                log.error(String.format("Error starting Local Transfer server %s", e));
+                log.error(String.format("Error starting Local Transfer server %s", e), e);
+                e.printStackTrace();
```

Comment:
wasnt log logging exceptions? We are using the format where exception should be logged